docs/html/select/select-with-icon.html

```diff
@@ -1,22 +1,25 @@
 <div class="ray-form-item">
   <div class="ray-select ray-select--with-icon-left">
-    <svg class="ray-select__icon--left" viewBox="0 0 25 25">
-      <g id="budicon-profile-picture">
-        <path
-          d="M12.5,4A4.5,4.5,0,1,0,17,8.5,4.5,4.5,0,0,0,12.5,4Zm0,8A3.5,3.5,0,1,1,16,8.5,3.504,3.504,0,0,1,12.5,12Zm0-12A12.4886,12.4886,0,0,0,5.0007,22.4834v0a12.4325,12.4325,0,0,0,14.9983,0v0q.5-.3761.9593-.7988l0,0A12.4869,12.4869,0,0,0,12.5,0Zm0,24a11.4432,11.4432,0,0,1-7.3931-2.7041,7.4887,7.4887,0,0,1,14.7863,0A11.4432,11.4432,0,0,1,12.5,24Zm8.25-3.5061a8.4871,8.4871,0,0,0-16.5,0,11.5,11.5,0,1,1,16.5,0Z"
-        />
-      </g>
-    </svg>
-    <select class="ray-select__input">
-      <option value="" disabled selected data-ray-placeholder></option>
-      <option value="Pikachu">WeWork</option>
-      <option value="Squirtle">WeLive</option>
-      <option value="Bulbasaur">WeGrow</option>
-      <option value="Charmander">WeTech</option>
-    </select>
-
-    <label class="ray-select__label">
-      Better Together
-    </label>
+    <div class="ray-select__icon--left">
```

Comment:
BREAKING CHANGE: the (svg) icon no longer receives the `ray-select__icon--left` class. There is now a wrapping div around the svg icon.